app/src/main/java/chat/rocket/android/userdetails/presentation/UserDetailsPresenter.kt

```diff
@@ -118,4 +120,28 @@ class UserDetailsPresenter @Inject constructor(
             }
         }
     }
+
+    fun utcToTime(utcOffset:Float?): String {
+        val gmtTime = System.currentTimeMillis() // 2.32pm NZDT
+        var timezoneAlteredTime: Long
```

Comment:
You could Lift out the assignment `timeZoneAlteredTime`. Directly could 
`timeZoneAlteredTime = if (condition ) value else other value`.
This would be according to Kotlin Coding style :)